### ISSUE TITLE
Missing details in tags with Pytorch

### ISSUE THREAD

**Issue by Pozimek** [j3soon/tbparse#9]
My Pytorch Tensorboard logs have been created using the following function calls:

`self.writer.add_scalars('Loss (Detailed)/Training', {
                        "Train_loss":loss_t.val}, iteration)
                    self.writer.add_scalars('Accuracy (Detailed)/Training', {
                        "Train_acc":accs.val}, iteration)
                    self.writer.add_scalars('Partial Losses/Training',{
                            "Reward_train" : R.val,
                            "Baseline_train" : B.val,
                            "Base_Loss_train" : loss_base.val,
                            "Class_Loss_train" : loss_class.val}, iteration)`

Tbparse does not recognize the finer tags and only returns data organized under the top-level tags, i.e. instead of organizing the data under the following 4 unique tags:

> Partial Losses/Training_Reward_train
> Partial Losses/Training_Baseline_train
> Partial Losses/Training_Base_Loss_train
> Partial Losses/Training_Class_Loss_train

It combines all of the data under the top-level "Partial Losses/Training" tag. As a result all of my partial losses, as well as my validation and training data are mixed up.

**Comment by j3soon** [comment]
**TL;DR**: Specify `dir_name` as an extra column:

```py
reader = SummaryReader(log_dir, extra_columns={'dir_name'})
```

See [this page](https://tbparse.readthedocs.io/en/latest/pages/parsing-scalars.html#load-log-directory) for further details.

<hr>

The PyTorch implementation of `add_scalars` does not store the `main_tag`s as tensorboard tags. Instead, it stores the event files under a directory named by the `main_tag`. More specifically, the following code:

```py
writer = SummaryWriter(log_dir)
writer.add_scalars('Loss (Detailed)/Training', { "Train_loss": 1}, 0)
writer.add_scalars('Accuracy (Detailed)/Training', { "Train_acc": 2}, 0)
writer.add_scalars('Partial Losses/Training', { "Reward_train" : 3, "Baseline_train" : 4, "Base_Loss_train" : 5, "Class_Loss_train" : 6}, 0)
```

results in the following events in the file system:

```
run
├── Accuracy (Detailed)_Training_Train_acc
│   └── events.out.tfevents.1652161869.ubuntu.27126.2
├── events.out.tfevents.1652161869.ubuntu.27126.0
├── Loss (Detailed)_Training_Train_loss
│   └── events.out.tfevents.1652161869.ubuntu.27126.1
├── Partial Losses_Training_Baseline_train
│   └── events.out.tfevents.1652161869.ubuntu.27126.4
├── Partial Losses_Training_Base_Loss_train
│   └── events.out.tfevents.1652161869.ubuntu.27126.5
├── Partial Losses_Training_Class_Loss_train
│   └── events.out.tfevents.1652161869.ubuntu.27126.6
└── Partial Losses_Training_Reward_train
    └── events.out.tfevents.1652161869.ubuntu.27126.3
```

I assume you used the following code to read the events:

```py
reader = SummaryReader(log_dir)
df = reader.scalars
```

resulting in the mixed up events:

  | step | tag | value
-- | -- | -- | --
0 | 0 | Accuracy (Detailed)/Training | 2.0
1 | 0 | Loss (Detailed)/Training | 1.0
2 | 0 | Partial Losses/Training | 5.0
3 | 0 | Partial Losses/Training | 4.0
4 | 0 | Partial Losses/Training | 6.0
5 | 0 | Partial Losses/Training | 3.0

You should specify `dir_name` as an extra column to differentiate the main tags:

```py
reader = SummaryReader(log_dir, extra_columns={'dir_name'})
df = reader.scalars
```

resulting in the expected events:

  | step | tag | value | dir_name
-- | -- | -- | -- | --
0 | 0 | Accuracy (Detailed)/Training | 2.0 | Accuracy (Detailed)_Training_Train_acc
1 | 0 | Loss (Detailed)/Training | 1.0 | Loss (Detailed)_Training_Train_loss
2 | 0 | Partial Losses/Training | 5.0 | Partial Losses_Training_Base_Loss_train
3 | 0 | Partial Losses/Training | 4.0 | Partial Losses_Training_Baseline_train
4 | 0 | Partial Losses/Training | 6.0 | Partial Losses_Training_Class_Loss_train
5 | 0 | Partial Losses/Training | 3.0 | Partial Losses_Training_Reward_train
